dev-util/scons/scons-4.3.0.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{8..10} )
+PYTHON_REQ_USE="threads(+)"
+
+MY_P="SCons-${PV}"
+
+inherit distutils-r1 multiprocessing
+
+DESCRIPTION="Extensible Python-based build utility"
+HOMEPAGE="https://www.scons.org/"
+SRC_URI="
+	https://downloads.sourceforge.net/project/${PN}/${PN}/${PV}/${MY_P}.tar.gz -> ${P}.tar.gz
```

Comment:
@mgorny renaming it maintains support for both scons and SCons and do not require any further changes on ebuild